proposals/apple-mdm/base-setup-clix.md

```diff
@@ -0,0 +1,205 @@
+# MDM Base Setup
+
+Note: these can all be namespaced behind something like `apple-mdm` if needed. But I would like to avoid creating an mdm vertical vs. an osquery vertical to create an integrated product.
+
+## APNS
+### Step 1 - Generate CSR
+``` sh
+$ fleetctl setup apns init
+Generating MDM keys at ./
+  fleet-apns-pki.crt ... done
+  fleet-apns-pki.key ... done
+  fleet-apns-push.key ... done
+  fleet-apns-push.csr ... done
+Send the .csr and .crt files to your friendly Fleet representative
+
+Keep the .key files. You will need them to complete this process. 
+
+Fleet will return a signed certificate to you within 1 business day. Use that file with the .key files in the `fleetctl setup apns gen-req` command.
```

Comment:
> with the .key files

Just the `fleet-apns-pki.key` is needed for the `gen-req` step.

Reply:
will change